src/libraries/pkg/test/testPackages.proj

```diff
@@ -3,19 +3,26 @@
 
   <ItemGroup>
     <PackagesToTest Condition="'$(PackagesToTest)' != ''" Include="$(PackagesToTest)" />
-    <!-- transport package not expected to be closure complete  -->
-    <ExcludePackages Include="Microsoft.AspNetCore.Internal.Transport" />
-    <ExcludePackages Include="$(ExcludePackages)" />
 
     <PackageReports Condition="'@(PackagesToTest)' == ''" Include="$(PackageReportDir)*.json" Exclude="@(ExcludePackages->'$(PackageReportDir)%(Identity).json')"/>
     <PackageReports Condition="'@(PackagesToTest)' != ''" Include="@(PackagesToTest->'$(PackageReportDir)%(Identity).json')" />
 
     <!-- support override via commandline -->
     <RuntimesToInclude Condition="'$(RuntimesToInclude)' != ''" Include="$(RuntimesToInclude)" />
     <TargetFrameworksToInclude Condition="'$(TargetFrameworksToInclude)' != ''" Include="$(TargetFrameworksToInclude)" />
+    <SymbolsPackages Include="$(LibrariesAllConfigShippingPackagesDir)*.symbols.nupkg" />
+    <AllPackagesFilePath Include="$(LibrariesAllConfigShippingPackagesDir)*.nupkg" Exclude="@(SymbolsPackages)" Condition="'$(SymbolsPackages)' != ''">
```

Comment:
What's this about `Condition="'$(SymbolsPackages)' != ''"`?

Reply:
This is avoid evaluating the substring functions when we dont have any packages eg during restore